src/DynamoCore/Graph/Workspaces/SerializationExtensions.cs

```diff
@@ -29,7 +29,8 @@ internal static string ToJson(this WorkspaceModel workspace, EngineController en
                 Formatting = Formatting.Indented,
                 Converters = new List<JsonConverter>{
                         new ConnectorConverter(),                        
-                        new WorkspaceWriteConverter(engine)
+                        new WorkspaceWriteConverter(engine),
+                        new DummyNodeWriteConverter()
```

Comment:
what happens with view serialization, does the nodeView of a dummy node also get serialized correctly? I think it would be good to have minimal automated testing thats interact with dummy nodes. That will take some messing about because the serialization tests ignore graphs that load up with dummy nodes.

Reply:
The view serialization appears to work as expected, but there needs to be some more testing done. I will look into this more closely.